src/installer/pkg/packaging/rpm/pack-deps.proj

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project='package.props' />
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(InstallerTasksAssemblyPath)"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(InstallerTasksAssemblyPath)"/>
+
+  <PropertyGroup>
+    <TargetFramework>$(NETCoreAppFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <Target Name="GenerateSingleRuntimeDependenciesRpm">
```

Comment:
Could you leave this target where it was in `package.targets` and have the `DepsProjects` items use `$(MSBuildProjectFullPath)`? Would reduce the diff between the RPM vs. Deb targets, I think.

Reply:
Will check the feasibility.

Reply:
Fixing this in new commit.